joomla/joomla-cms
# Add missing language item for com_menus

Pull Request for Issue # .

### Summary of Changes
This PR just add missing language item COM_MENUS_TYPE_SYSTEM to com_menus

### Testing Instructions
1. Install Joomla 4.0
2. Try to create a menu item
3. Click on Select button in **Menu Item Type** to select a menu option

Before patch, you see the text **COM_MENUS_TYPE_SYSTEM** in the popup. After patch, the text is replaced with System Links (like in Joomla 3)

### Documentation Changes Required
None
